src/DashboardView/Settings/EventTypes/EventTypes.js

```diff
@@ -52,9 +53,15 @@ class EventTypes extends Component {
         this.fetchEventTypes();
     }
 
-    delteEventType(eventType) {
+    deleteEventType(eventType) {
+        $("#delete-prompt-modal").modal('toggle');
         this.props.dataSender.deleteEventType(eventType, this.deletionSuccessful, this.receiveError);
     }
+
+    /* istanbul ignore next */
+    onAbort() {
```

Comment:
Please rename this method to `handleAbortDeleteEventType` or `handleAbortDelete`, because it's the base method of a callback.